misk/web/tabs/README.md

```diff
@@ -133,40 +133,70 @@ In your service's project `build.gradle` file you will need to add the following
   }
   
   ...
-  ext.generateDockerContainerName = { service, project, task ->
-    return "${new Date().format("YMD-HMS")}-${service}-${project}-${task}"
-  }
+  task web(type: Exec) {
```

Comment:
Again seems like drift/managing this in two spots might become an issue. Consider just saying 'see this file'?